tsconfig.config.json

```diff
@@ -8,6 +8,7 @@
   "compilerOptions": {
     "moduleResolution": "Bundler",
     "composite": true,
+    "resolveJsonModule": true,
```

Comment:
Add the config only in one file, not both.
Only in tsconfig.app.json

Remove this change